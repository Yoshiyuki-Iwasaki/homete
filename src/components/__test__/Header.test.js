import { render } from "@testing-library/react";
import React from "react";
import Sample from "../Header";

jest.mock("next/router", () => ({
  useRouter() {
    return {
      asPath: "/",
    };
  },
}));

describe("Navコンポーネント 初期表示の確認", () => {
  test("サンプルページ", () => {
    const { asFragment } = render(<Sample />);
    expect(asFragment()).toMatchSnapshot();
  });


  // test("check nav link", () => {
  //   const doc = render(<Sample />);
  //   const inputElement = doc.getByTestId('LinkToMypage');

  //   expect(inputElement.getAttribute('href')).toBe('/mypage');
  // });
});