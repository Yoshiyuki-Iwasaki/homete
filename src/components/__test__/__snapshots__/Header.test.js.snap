// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navコンポーネント 初期表示の確認 サンプルページ 1`] = `
<DocumentFragment>
  <h6>
    Loading...
  </h6>
</DocumentFragment>
`;
